src/api/ledger/parse/trustline.js

```diff
@@ -15,8 +15,10 @@ function parseFlag(flagsValue, trueValue, falseValue) {
   return undefined;
 }
 
-function parseQuality(quality) {
-  return (new BigNumber(quality)).shift(-9).toNumber();
+function parseQuality(quality?: number) {
+  if (typeof quality === 'number') {
```

Comment:
This function is called with data that comes from server, so flow really can't check anything here.